src/main/java/mate/academy/internetshop/controllers/CartViewController.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.internetshop.controllers;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.model.Product;
+import mate.academy.internetshop.model.ShoppingCart;
+import mate.academy.internetshop.service.ShoppingCartService;
+
+@WebServlet("/cart")
+public class CartViewController extends HttpServlet {
```

Comment:
```suggestion
public class GetShoppingCartController extends HttpServlet {
```

Reply:
I renamed it to view as it's convenient for me to understand what it does, that I can view my cart through this method

Reply:
Done